src/components/common/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I'm not very familiar with prop-types library. Some comments might be good to explain this libraries usage.